test/api-method-documentation.test.js

```diff
@@ -520,13 +556,13 @@ describe('<api-method-documentation>', function() {
         it('sets URL from base uri', async () => {
           const element = await baseUriFixture(amf, endpoint, method);
           await aTimeout();
-          assert.equal(element.endpointUri, 'https://domain.com/people/{personId}');
+          assert.equal(element.endpointUri, 'http://{instance}.domain.com/v1/people/{personId}');
```

Comment:
These tests shows that this change breaks the component. The expected result of setting the baseUri property is that the rendered URI uses the `baseUri` instead of the one defined in the API model. This fixture has `baseuri="https://domain.com"` attribute set therefore the computed endpointUri should be `https://domain.com/people/{personId}`.